packages/remirror__theme/src/extension-tables-theme.ts

```diff
@@ -2,6 +2,15 @@ import { css } from '@linaria/core';
 
```

Comment:
highlight for reviewers: I added all the style for `@remirror/extension-react-tables` into `extension-tables-theme.ts`. Should I create a new file to handle this?

Reply:
Looks good